web3_proxy/src/frontend/users/stats.rs

```diff
@@ -53,8 +53,22 @@ pub async fn user_revert_logs_get(
         .await
         .web3_context("failed loading user's key")?;
 
+    // Also add rpc keys for which this user has access
+    let shared_rpc_keys = secondary_user::Entity::find()
+        .filter(secondary_user::Column::UserId.eq(user.id))
+        .all(db_replica.conn())
+        .await?
+        .into_iter()
+        .map(|x| x.rpc_secret_key_id);
```

Comment:
the frontend team asked that we also keep track of the secondary_user's permissions. that way they can show/hide various elements in the UI.

Reply:
we do keep track of secondary user permissions, i will filter the map accordingly